cli/tests/e2e/snapshots/test_baseline/test_crisscrossing_merges/baz-bar/stderr.txt

```diff
@@ -32,7 +32,18 @@ Switching repository to baseline commit 'bar'.
     --baseline-commit=$(git merge-base bar HEAD)
 
 Scanning 1 file.
-
+                                                                                
+                                                                                
+╭─────────╮                                                                     
+│ Results │                                                                     
+╰─────────╯                                                                     
+                                                                                
```

Comment:
@ievans this is just the stderr stream, the stdout (which contains actual results) is snapshotted in a different .txt file.

In your terminal or CI logs you'd see both streams together and it would not look broken.

If there are no results, we do not print the heading.

Reply:
got it great thanks!